.travis.yml

```diff
@@ -2,30 +2,25 @@ dist: xenial
 language: python
 python:
 - 3.6
-cache:
-- pip
-- apt
+env:
+  global:
+  - secure: KY3stltR4M1obId8R1s98gUYdxCqZdfnUMJ+5qRiXIdMy+ZsyHlZ/cVtjpIqPBkmDVXRpC9sp66j7ipNl/WEA1Su5mkQSuec/h2d9aN57MrzXClPs4qSadyCvOfA8wjfKbuw6eTBvX6WF7UA6eC4nv07q+sAp2CuaMRm8fi5jbk9fcF+3QCFmd5ORuBXoVBRnU3lvsf3ih8l0ESxFfgjAfsuDoXYV9/8eyhknn9rpSHuX1lP0Ut/+Cs40bF8M3ujknA8GfdkMF9dRdSOvqyX7Jz4A/AiXKG69ZJXxR1XdZuZmdlMytjBw/XrSjNm/m2/gYca+MAPGA1/FEO20yf7a9VItJVBppO+QJ6QFZ+gB0vz7lRy88grAk1iFwi0+kMmXgcInjlb+iStw+MIbdBTzPW7AFkIEDqyN5XKhJgah8cFvYE0GOP/ROXr3AgBEvT3yj7kO+fHE7bmCkpjEaTIKkosb+/jTO3ty8YWHjT7NVaNFmX5i6gxKa3jbiSLtPkEsByNZOYtuIOWIkvnLOVNiC9Y9E6ZN0F+qiKinuDYqvmxx+HvHhJmjO+/E5DSh0cqgN3OFYUB/1kVEI/B+MaX/KbVdAA8KCdXvaClURcbBSZlD4nqOsGUcDmGbri/lMmG/9ozVrmlwyfZ6cfRapmBRouC6EcQ2FGLqZe4r3LYngc= # pragma: allowlist secret
 install:
-- pip install pipenv
-- pipenv install --dev --deploy
-- pipenv graph
+- 'curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python'
+- source $HOME/.poetry/env
+- which poetry
+- poetry install -vv
 script:
-- pipenv run pytest -vv --cov=gen3utils --cov-report xml ./tests
-after_script:
-- pipenv run python-codacy-coverage -r coverage.xml
+- poetry run pytest -vv ./tests
```

Comment:
Not sure how much Codacy is used, but I noticed this in the [latest build of master](https://travis-ci.com/github/uc-cdis/gen3utils/builds/174107901#L398): 
```
$ pipenv run python-codacy-coverage -r coverage.xml
2020-07-02 19:03:43,366 - ERROR - environment variable CODACY_PROJECT_TOKEN is not defined.
```

Reply:
Looks like Codacy is not installed for uc-cdis org. Let's worry about this laterr.